tests/integration/test_models.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+import os
+os.environ['DJANGO_SETTINGS_MODULE'] = 'test_project.settings'
```

Comment:
With that, and no `django.setup()`, I'm getting a

    tests\integration\test_models.py:11: in <module>
        'secret': 'demoprivatekey'})
    .tox\py27-django18\lib\site-packages\django\conf\__init__.py:58: in configure
         raise RuntimeError('Settings already configured.')
    E   RuntimeError: Settings already configured.

when run with `tox -e py27-django18` and

    AppRegistryNotReady: The translation infrastructure cannot be initialized before the apps registry is ready. Check that you don't make non-lazy gettext calls at import time.

when run from a PyCharm unit test run configuration.